frontend/src/layout/navigation/TopBar/announcementLogic.ts

```diff
@@ -17,6 +17,7 @@ export enum AnnouncementType {
 
 // Switch to `false` if we're not showing a feature announcement. Hard-coded because the announcement needs to be manually updated anyways.
 const ShowNewFeatureAnnouncement = false
+const ShowAttentionRequiredBanner = false
```

Comment:
`ShowAsyncMigrationRequiredBanner`?

Reply:
Well I guess things are already a bit confusing here - we check `!asyncMigrationsOk` but return `AnnouncementType.AttentionRequired`

I went for the more general switch like above